python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -463,11 +463,15 @@ def _keep_min_worker_of_node_type(
             node_type_counts(Dict[NodeType, int]): The non_terminated node
```

Comment:
Can we change the name of this function to `_keep_worker_of_node_type` and then modify the description to say we keep workers to respect the min_owkrers constraint but we terminate workers if the node type has been deleted from the cluster config.